RSEConUK2019-Cylc-Talk/index.html

```diff
@@ -583,118 +801,279 @@ <h2 style="text-align: right">Example context: <em>NWP</em></h2>
+              https://commons.wikimedia.org/wiki/File:LaTeX_logo.svg
+            -->
+            <img src="media/external-logos/latex-logo.png" alt="latex-logo"
+            style="height:50px;">
+          </td>
+          <td>
+            <span>&#10230;</span> 
+          </td>
+          <td>
+            <!--
+              Source (fair usage):
+              http://www.sphinx-doc.org/en/master/_static/sphinxheader.png
+            -->
+            <img src="media/external-logos/sphinx-logo.png" alt="sphinx-logo"
+            style="height:50px;">
+          </td>
+        </tr>
+      </table>
     </div>
 
-    <div class="step">
-      <b>Integrated</b> browser-based
-      <a href="https://github.com/cylc/cylc-ui/files/3362486/design.document.pdf">design</a>...
-      <img src="media/compound-visuals/cylc-8-mockup.png" alt="cylc-8-mockup"
-        style="height:550px;">
-      <b>Improved UX</b> after user feedback
+    <div
+      id="new-web-UI"
+      class="step full vflex"
+      >
+      <p><em>Integrated</em>, <em>browser-based</em>
+      <a href="https://github.com/cylc/cylc-ui/files/3362486/design.document.pdf">design</a>...</p>
+      <img
+	 src="media/compound-visuals/cylc-8-mockup.png" alt="cylc-8-mockup"
+	 style="width:80%;display:block;margin-left:auto;margin-right:auto">
+
+      <p> with feedback-led <em>improved UX</em></p>
+          </div>
+    <!--
+         Section "4": Closing Slides (not really a section, but strictly this
+         final slide is not on the future of Cylc, so put on another level as
+         we ease into the closing slides which then go off diagonally.
+    -->
+    <div
+      id="the-present"
+      class="step"
+      data-rel-y="1h"
+      data-rel-to="future-cylc-8"
+      >
+      The <b>Present</b>: for now...
     </div>
 
-    <div class="step">
+    <div
+      id="how-to-install"
+      class="step vflex"
+      data-rel-x="1w"
+      data-rel-y="0"
+      >
       <ul>
-        <li><b>Cylc-8.0.0 "coming soon"</b> (<i>optimistic</i> estimate for
+        <li><em>Cylc-8.0.0 "coming soon"</em> (<i>optimistic</i> estimate for
           release ~May 2020)
         </li>
-        <br>
-        <li><b>Until then, please use Cylc 7.8.<i>X</i></b>, e.g. the latest,
-          <a href="https://github.com/cylc/cylc-flow/releases/tag/7.8.3">
-          7.8<i>.3</i></a>, which is thoroughly tested & supported
+        <li>Until then, <em>please use Cylc 7.8.<i>X</i></em> (the
+	  latest, currently
+	  <a href="https://github.com/cylc/cylc-flow/releases/tag/7.8.4">
+          7.8<i>.4</i></a>) which is thoroughly tested & supported
         </li>
-        <br>
         <li>Install for Linux
           <a href="https://github.com/cylc/cylc-flow/blob/7.8.x/INSTALL.md">
           via GitHub</a>
         </li>
       </ul>
     </div>
 
-
-    <!-- Closing Slides -->
     <div
       id="esiwace-funding"
       class="step"
       data-rel-y="1h"
```

Comment:
Also add `data-rel-x=0` to stop these slides going diagonal.

Reply:
That was intentional, as they seem like unrelated closing slides, so making them pan diagonally to almost "pull away" from the proper sections for the end of the talk, contrasting to the horizontal movements inside sections, seemed nice. But it seems you are not keen so I'm happy to keep the closing section pan horizontally as for the true sections.